src/test/scala/nl/hugo/redbook/ch6/CNG.scala

```diff
@@ -0,0 +1,24 @@
+package nl.hugo.redbook.ch6
+
+object CNG {
```

Comment:
Maybe my days as a Gamer has primed me to read RNG as 'Ranger' ;) Somehow, CNG turns into 'Conga' in my head. I guess naming is still one of the hardest things in programming?